README.md

```diff
@@ -2,13 +2,43 @@
 
 Icons, fonts, colors, components and all user interface items for OSRD project.
 
-## TODO
+## Development
 
-We should write a README to explain the goal of the `osrd-ui` repository and the design system as
-the whole.
+Development will leverage preconstruct. **Documentation yet to come**.
 
-We should also explain how it is related to the `osrd-design` repository.
+## Publishing versions
 
-## Credits
+In a nutshell: we consider our version numbers metadata of the project. We use [Semantic Versioning](https://semver.org/).
 
-### ui-icons
+### Implications
+
+We do not change the version on local package versions, we keep the file to the version `0.0.1-dev` which is not a real version and can be easily identified as a development version.
+
+The version numbers for our releases are solely managed through git tags. It implies that when we update a single package, we release all the project together : even if there are no changes between two release (let's say we update ui-icons but nothing else, we would make a release that would publish all packages anyway).
+
+It's the tradeoff we make to keep the project simple and easy to manage, and to avoid the complexity of managing multiple compatible versions of the project.
+
+In summary : we consider a version number as being an indivisible release of all the subpackages of the repository.
+
+### Making a release
+
+1. Create a annotated git tag (let's say `0.0.30`, you would do a `git tag -a 0.0.30`) on the `dev` branch and push it. **Note: we do not tag with a V in front of the version number, on our example the tag would be 0.0.30 and not v0.0.30**.
+2. Create a github release (By convention, we use the same name as the tag with the letter `v` in front of it, `v0.0.30`. You can give fancy names to release if you want to : `0.0.30 : Camembert`, it will only appear in the release page of GitHub).
```

Comment:
Ah yes, a completely random name, [totally not a disguised proposal for the next PI iteration name](https://en.wikipedia.org/wiki/Foreshadowing).

Reply:
already done... good try!

Reply:
😇 I'm pleading the fifth.